extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalFooter_index.php

```diff
@@ -11,6 +13,18 @@
 					) ?></span>
 			</h2>
 		</div>
+	<?php elseif ( isset ( $model['header'] ) ) : ?>
+		<h2 class="wds-global-footer__header">
+			<a href="<?= Sanitizer::encodeAttribute( $model['header']['href'] ); ?>"
+			   data-tracking-label="<?= Sanitizer::encodeAttribute( $model['header']['title']['key'] ) ?>"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)